src/main/java/com/github/manolo8/darkbot/core/objects/gui/AssemblyGui.java

```diff
@@ -1,4 +1,22 @@
 package com.github.manolo8.darkbot.core.objects.gui;
 
-public class AssemblyGui {
+import com.github.manolo8.darkbot.core.objects.Gui;
+import eu.darkbot.api.API;
+import eu.darkbot.api.managers.BotAPI;
+import eu.darkbot.util.Timer;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class AssemblyGui extends Gui implements API.Singleton {
+    private final BotAPI bot;
+    private final Timer guiUsed = Timer.getRandom(19_000, 1000);
```

Comment:
Is anmything at all updating this? feels like this pr is missing adding an actual API ontop of assembly that would refresh this timer

Reply:
added the manager to interact with the gui